modules/spo-base-syscalls.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * security/security_profiles_operator/spo-advanced.adoc
+
+:_content-type: PROCEDURE
+[id="spo-base-syscalls_{context}"]
+= Base syscalls for a container runtime
+
+You can use `runc` to establish the minimum required syscalls for a given runtime to start a container.
+
+.Procedure
+
+* Edit the `SeccompProfile` kind object and add `baseProfileName: runc-v1.0.0` to the `spec` field:
+
+[source,yaml]
+----
+apiVersion: openshift-security-profiles.x-k8s.io/v1beta1
```

Comment:
@jhrozek I see v1beta1, v1alpha1, v1alpha2.... any idea what version will be used at launch time yet? Also, will it change from `x-k8s.io`?

Reply:
seccomp stays at v1beta1, selinux is at v1alpha2, most of the other APIs are at v1alpha1.
The API group will stay at `x-k8s.io` (not that OCP still uses a lot of APIs that end with `coreos.com` ...)
We will upgrade all API versions to something more sane (we did get some flak from OCP architects about shipping v1alpha1 APIs...) but we didn't make it for the current release. THe APIs still need a bit more time to stabilize upstream, even though upstream tends to only change in a backwards-compatible manner.